src/bin/deku_node.ml

```diff
@@ -189,14 +189,16 @@ let node folder prometheus_port =
 (* TODO: https://github.com/ocaml/ocaml/issues/11090 *)
 let () = Domain.set_name "deku-node"
 
-let () =
+let setup_log level () =
   (* This is needed because Dream will initialize logs lazily *)
   Dream.initialize_log ~enable:false ();
-  match Sys.getenv_opt "DEKU_LOGS" |> Option.map String.lowercase_ascii with
-  | Some "json" ->
-    Logs.set_reporter (Json_logs_reporter.reporter Fmt.stdout);
-    Logs.set_level (Some Info)
-  | _ -> Logs.set_reporter (Logs.format_reporter ~app:Fmt.stdout ())
+  (match Sys.getenv_opt "DEKU_LOGS" |> Option.map String.lowercase_ascii with
+  | Some "json" -> Logs.set_reporter (Json_logs_reporter.reporter Fmt.stdout)
+  | _ -> Logs.set_reporter (Logs.format_reporter ~app:Fmt.stdout ()));
+  Logs.set_level level;
+  ()
+
+let setup_log = Term.(const setup_log $ Logs_cli.level ())
```

Comment:
I don't think we need both environment variables and command line options so this can get back to what it was before, @ulrikstrid?

Reply:
I'm happy with only 1 way of setting it